client/src/store/profile/common.js

```diff
@@ -0,0 +1,7 @@
+const ActionType = Object.freeze({
```

Comment:
```suggestion
const ActionType = {
```
let's avoid it this time. most of enums in the app just a plain object